src/Loader.php

```diff
@@ -473,7 +491,7 @@ public static function inject_after_notices() {
 	 * Edits Admin title based on section of wc-admin.
 	 *
 	 * @param string $admin_title Modifies admin title.
-	 * @todo Can we do some URL rewriting so we can figure out which page they are on server side?
+	 * TODO: Can we do some URL rewriting so we can figure out which page they are on server side.
```

Comment:
Nitpick: We should change this to our standardize format of `@todo`:
```suggestion
	@todo Can we do some URL rewriting so we can figure out which page they are on server side?
```